x-pack/plugins/task_manager/server/saved_objects/migrations.ts

```diff
@@ -37,6 +37,10 @@ export function getMigrations(): SavedObjectMigrationMap {
       pipeMigrations(alertingTaskLegacyIdToSavedObjectIds, actionsTasksLegacyIdToSavedObjectIds),
       '8.0.0'
     ),
+    '8.1.0': executeMigrationWithErrorHandling(
```

Comment:
Should be 8.2.0?

Reply:
Yes... this was before 8.1 FF. 
Fixed, thanks.